shoulda_matchers.rb

```diff
@@ -0,0 +1,6 @@
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :rspec
+    with.library :rails
+  end
+end
```

Comment:
Did this file get added by mistake? I see it is already in `spec/support`.

Reply:
whoops.  I thought I'd added that file but it was clearly in the wrong dir.